src/test/java/data/CameraShotTest.java

```diff
@@ -0,0 +1,34 @@
+package data;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Bart.
+ */
+public class CameraShotTest {
+    CameraShot cameraShot;
+    @Before
+    public void initialize() {
+        cameraShot = new CameraShot("camerashot-1");
+    }
+
+    @Test
+    public void getName() throws Exception {
+        assertEquals("camerashot-1", cameraShot.getName());
+    }
+
+    @Test
+    public void getInstance() throws Exception {
+        assertEquals(0, cameraShot.getInstance());
```

Comment:
The `getInstance()` method should be tested by adding several cameras, 2-3. Since the `instanceCounter` variable is initialized at `0`.